cmd/machine-healthcheck/main.go

```diff
@@ -24,6 +24,7 @@ func printVersion() {
 
 func main() {
 	watchNamespace := flag.String("namespace", "", "Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.")
+	metricsPort := flag.String("metrics-port-healthcheck", "8081", "Port at which metrics for machine healthchecks will be running")
```

Comment:
I don't think we need to name these differently across each binary, `metrics-port` should be ok I think

This is more than just a port though, IIRC, it should be of the format `:8081` as the default because it's the address (ie you could serve metrics on localhost only `127.0.0.1:8081`), so maybe it should be `metrics-address` instead?

Reply:
You are right, going to change it to `MetricsBindAddress` instead.